### DISCUSSION TITLE
How to enable trickplay for Jellyfin 10.10.1 for web interface?

### DISCUSSION THREAD

**Discussion by lamyergeier** [jellyfin/jellyfin#12952] [Q&A] [answered]
I did the following but when I move my mouse over a video thumbnail in the web interface nothing happens! 

Does the following would generate it for all existing video files? If not, how to do it.

# Trickplay Settings

![image](https://github.com/user-attachments/assets/701dbd48-190d-485f-8ab4-2c756912a21f)

![image](https://github.com/user-attachments/assets/29f41a63-da92-48a1-8aa8-662b60c19a2f)

# Setup - Docker

```bash
			dVideoP='/mnt/Media/Videos'
			uid="$(id -u)"
			gid="$(id -g)"

			docker pull jellyfin/jellyfin

			docker volume create jellyfin-config
			docker volume create jellyfin-cache

			docker run -d \
				--name jellyfin \
				--user "${uid}:${gid}" \
				--net=host \
				--volume jellyfin-config:/config \
				--volume jellyfin-cache:/cache \
				--mount type=bind,source="${dVideoP}/Library",target=/Library,readonly \
				--mount type=bind,source="${dVideoP}/YouTube",target=/YouTube,readonly \
				--restart=unless-stopped \
				jellyfin/jellyfin
```

**Comment by crobibero** [comment, marked as answer]
Did you enable TrickPlay in the library settings?
![image](https://github.com/user-attachments/assets/02e4efb9-047c-497e-b535-0599346d724d)
